### PR TITLE
docs: fix of the site nav in tokens

### PR DESCRIPTION
**Pull request by yorbi-dp**
# docs: fix of the site nav in tokens

## Obligatory GIF (super important!)

Obligatory GIF

## :hammer_and_wrench: Type Of Change

These types will increment the version number on release:

- [ ] Fix
- [ ] Feature
- [ ] Performance Improvement
- [ ] Refactor

These types will not increment the version number, but will still deploy to documentation site on release:

- [x] Documentation
- [ ] Build system
- [ ] CI
- [ ] Style (code style changes, not css changes)
- [ ] Test
- [ ] Other (chore)

## :book: Jira Ticket

https://dialpad.atlassian.net/browse/DLT-1512

## :book: Description

- Small fix on the Tokens Site Navbar.

## :pencil: Checklist

For all PRs:

- [x] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [x] I have reviewed my changes.
- [x] I have added all relevant documentation.
- [x] I have considered the performance impact of my change.

### CODE REVIEW

**Comment by juliodialpad** [comment]
Screenshot 2024-01-26 at 10 31 29 a m

You only need to check one option there, and should match with your PR title, as your PR title starts with (docs), you should only check the "Documentation" checkbox

**Comment by yorbi-dp** [comment]
> You only need to check one option there, and should match with your PR title, as your PR title starts with (docs), you should only check the "Documentation" checkbox

Oh, got you @juliodialpad! Thanks! 🙌

File: packages/dialtone-icons/src/keywords.json [hunk 1]
```diff
@@ -2356,4 +2356,4 @@
       ]
     }
   }
-}
+}
```
Comment (juliodialpad) [inline packages/dialtone-icons/src/keywords.json:2359]: Remove the line at the end please

**Review by juliodialpad — CHANGES_REQUESTED** [review]

File: packages/dialtone-icons/src/keywords.json (hunk 1 above)
Comment (juliodialpad) [reply packages/dialtone-icons/src/keywords.json:2359]: Sorry seems like its only whitespace changes, not a new line.

**Review by juliodialpad — APPROVED** [review]
Working correctly, thanks!

File: packages/dialtone-icons/src/keywords.json (hunk 1 above)
Comment (yorbi-dp) [reply packages/dialtone-icons/src/keywords.json:2359]: Yeah! I noticed it had a line and removed it, that's why it's in the pr hah. 😄

**Pull request merged**